dashboards-observability/public/components/visualizations/charts/maps/heatmap.tsx

```diff
@@ -37,7 +37,9 @@ export const HeatMap = ({ visualizations, layout, config }: any) => {
     isEmpty(zMetrics) ||
     isEmpty(data[xaxisField.label]) ||
     isEmpty(data[yaxisField.label]) ||
-    isEmpty(data[`${zMetrics.aggregation}(${zMetrics.name})`])
+    isEmpty(data[`${zMetrics.aggregation}(${zMetrics.name})`]) ||
+    dataConfig?.dimensions.length > 2 ||
+    dataConfig?.metrics.length > 1
     // indexOf(NUMERICAL_FIELDS, zMetrics.type) < 0
```

Comment:
Do we need this commented line after the actual implementation?

Reply:
Eliminated this commented code